install_resources.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
Also it could automatically generate tests for all resources (see TODO below). Maybe it's possible to do the same with maven but I haven't looked into it yet.